### DISCUSSION TITLE
qemu adds 195 files to the shims directory

### DISCUSSION THREAD

**Discussion by amreus** [ScoopInstaller/Main#3076] [Q&A] [answered]
I think this is a case where it might make more sense to add qemu to the user path instead of polluting the shims folder. 

Is this acceptable, or something worth opening an issue for?

**Comment by rashil2000** [comment, marked as answer]
Seems reasonable. Can you make a PR?

**Reply by amreus** [reply]
Yep, I will. Thanks.
